src/stdune/path.mli

```diff
@@ -46,6 +46,14 @@ module Unspecified : sig
   type w = Path_intf.Unspecified.w
 end
 
+module Fpath : sig
```

Comment:
What does `Fpath` stands for?

Reply:
I moved `Fpath` to its own module and made it private. The idea is that it's supposed to be upcoming "new" path module that doesn't make any dune specific assumptions. Eventually, we'd rename `Fpath` to `Path`, and `Path` to `Dune_path` (or `Dpath`).

Reply:
Oh I see. This needs a comment explaining all this then, maybe even with a link to a ticket with the plan. Otherwise it's not obvious when reading the code. So the `F` stands for `Future` then?